### ISSUE TITLE
Compatibility with Simple Corn

### ISSUE THREAD

**Issue by faceofcat** [MatrexsVigil/harvestcraft#59]
it seems that after installing harvest craft you get weird behavior using bonemeal on the top part of a multiblock corn plant...

https://gfycat.com/flatsplendidgoldenmantledgroundsquirrel

Simple Corn mod: [ [curse forge link](https://minecraft.curseforge.com/projects/simple-corn?gameCategorySlug=mc-mods&projectID=246761) ] [ [github link](https://github.com/cleverpanda/SimpleCorn) ]

**Comment by richardhendricks** [comment]
ZOMG!  Leave in the feature, make the stalk climbable into a new Jack and the BeanStalk dimension!

**Comment by cleverpanda** [comment]
No. This is not an intended feature and people complain to me all the time. 
Pam, I believe this issue would apply to any crop taller than 1 block.
I think an exclusion list would be the best fix, as I'm handling right click harvesting myself, since it's a bit weird for corn.

**Comment by NicosaurusRex99** [comment]
Compile this mod, https://github.com/cleverpanda/SimpleCorn source is here if thats what you needed

**Comment by MelanX** [comment]
Can be closed, you merged the PR which fixes it.
